mvc/views/tag-helpers/authoring/sample/TagHlp/src/AuthoringTagHelpers/TagHelpers/z1EmailTagHelperCopy.cs

```diff
@@ -0,0 +1,15 @@
+﻿
```

Comment:
Seems like you semi-consistently have newlines at the beginning and end of your code files. Should try and keep these files as pristine as possible, especially since you call out to users to examine the code [here] (https://github.com/aspnet/Docs/pull/419/files#diff-e1203d81d2c6c6045ce634266844f635R17). Nuke beginning/end extra newlines and update references in rst. Will also make rst easier to maintain that way.